services/arrow-flight/src/test/java/com/dremio/service/flight/TestFlightSqlServerWithBasicAuth.java

```diff
@@ -28,7 +28,6 @@
 /**
  * Test FlightServer with basic authentication using FlightSql producer.
  */
-@Ignore
 public class TestFlightSqlServerWithBasicAuth extends AbstractTestFlightServer {
```

Comment:
Why was this change done as part of this PR, seems unrelated?

Reply:
That's because in the previous PR there was no FlightSql functionality yet, so this tests would fail